ci/tasks/build-stemcell-baremetal.yml

```diff
@@ -0,0 +1,11 @@
+---
+platform: linux
+image: docker:///mattcui/sl_stemcell
```

Comment:
Why is this image coming from your docker repo? Can we create a common one of the bosh-ecpi team?